doc/manual/src/protocols/derivation-aterm.md

```diff
@@ -0,0 +1,17 @@
+For historical reasons, [derivations](@docroot@/glossary.md#gloss-store-derivation) are stored on-disk in [ATerm](https://homepages.cwi.nl/~daybuild/daily-books/technology/aterm-guide/aterm-guide.html) format.
+
+Derivations either serialized in one of the following formats:
```

Comment:
```suggestion
Derivations are serialised in one of the following formats:
```

We try to use BE consistently.

Reply:
Ah and what I wrote was just pain ungrammatical too whoops.